.github/workflows/build-and-publish-images.yml

```
@@ -28,820 +28,715 @@ env:
-          echo "Found inbox SARIF: ${{ steps.scan-inbox.outputs.sarif }}"
-          if [ -n "$FOUND_FILES" ]; then
-            FOUND_FILES="$FOUND_FILES,${{ steps.scan-inbox.outputs.sarif }}"
-          else
-            FOUND_FILES="${{ steps.scan-inbox.outputs.sarif }}"
+          *🤖 Built from branch: \`${{ github.ref_name }}\`*
+          *📝 Commit: ${{ github.sha }}*
+          *⏰ Build run: ${{ github.run_number }}*
+          EOF
+
+      - name: Create GitHub Release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.12)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)